zipkin/propagation.go

```diff
@@ -23,17 +23,17 @@ import (
 	"github.com/uber/jaeger-client-go"
 )
 
-// Propagator is an Injector and Extractor
-type Propagator struct{}
+// ZipkinB3HTTPHeaderPropagator can be used to extract and inject Zipkin HTTP B3 headers into SpanContexts.
+type ZipkinB3HTTPHeaderPropagator struct{}
```

Comment:
type name will be used as zipkin.ZipkinB3HTTPHeaderPropagator by other packages, and that stutters; consider calling this B3HTTPHeaderPropagator

Reply:
Please advise on this as I'm not sure that we want to change the public interface (or if the repo conventions allow me to change the struct name to a different one than what is implied by the `New`)

Reply:
why change it at all? `zipkin.Propagator` seems fine, it's not like there's another "zipkin" thing.